editoast/src/converters/utils.rs

```diff
@@ -511,4 +514,55 @@ mod tests {
         assert_eq!(1., reference_coord(NodeId(0), &edge).lon);
         assert_eq!(0., reference_coord(NodeId(2), &edge).lon);
     }
+
+    #[test]
+    fn test_voltage() {
+        let edge = Edge {
+            id: "1".into(),
+            tags: HashMap::from([("voltage".into(), "15000".into())]),
+            ..Default::default()
+        };
+
+        let electrification = electrifications(&edge).unwrap();
+
+        assert_eq!(electrification.voltage, "15000V".into());
+    }
+
+    #[test]
+    fn test_voltage_no_redondant_v() {
+        let edge = Edge {
+            id: "1".into(),
+            tags: HashMap::from([("voltage".into(), "15000V".into())]),
+            ..Default::default()
+        };
+
+        let electrification = electrifications(&edge).unwrap();
+
+        assert_eq!(electrification.voltage, "15000V".into());
+    }
+
+    #[test]
+    fn test_multiple_voltages() {
+        let edge = Edge {
+            id: "1".into(),
+            tags: HashMap::from([("voltage".into(), "600;1500;3000;15000;25000".into())]),
+            ..Default::default()
+        };
+
+        let electrification = electrifications(&edge).unwrap();
+
+        assert_eq!(electrification.voltage, "600V".into());
+    }
+
+    #[test]
+    fn test_no_voltage() {
+        let edge = Edge {
+            id: "1".into(),
+            ..Default::default()
+        };
+
+        let electrification = electrifications(&edge);
+
+        assert!(electrification.is_none());
+    }
```

Comment:
We could replace this with a single parametrized test using `rstest`. Check the documentation here: https://docs.rs/rstest/latest/rstest/attr.rstest.html#test-parametrized-cases

Reply:
Done, or should I even try to fit the second one in the first one ?